client/app/telecom/telephony/line/calls/forward/telecom-telephony-line-calls-forward.controller.js

```diff
@@ -178,16 +178,32 @@ angular.module('managerApp').controller('TelecomTelephonyLineCallsForwardCtrl',
    * @return {Promise}
    */
   function loadAllOvhNumbers() {
-    return TelecomTelephonyLineCallsForwardService.loadAllOvhNumbers($stateParams.serviceName).then(
-      (allOvhNumbers) => {
-        self.allOvhNumbers = allOvhNumbers;
-        return allOvhNumbers;
-      },
-      (err) => {
-        TucToast.error($translate.instant('telephony_line_actions_line_calls_forward_number_load_error'));
-        $q.reject(err);
-      },
-    );
+    return TelecomTelephonyLineCallsForwardService.loadAllOvhNumbers($stateParams.serviceName)
+      .then(
+        (allOvhNumbers) => {
+          const numbers = allOvhNumbers.map((ovhNumber) => {
+            const filtered = self.listBillingAccounts.filter(billingAccount => billingAccount.description && billingAccount.billingAccount === ovhNumber.billingAccount);  // eslint-disable-line
```

Comment:
We try to avoid as much as possible disabling ESLint rules. It could be written like:

```suggestion
            const filtered = self.listBillingAccounts
              .filter(billingAccount => billingAccount.description
                && billingAccount.billingAccount === ovhNumber.billingAccount);
```

Reply:
Hello Antoine,
thanks for the tip, I didn't succeed when I tried to set proper multiline coding style, and every time I tried to commit, ESLint (I think) rewrote as a single line...
I updated my branch with this change.